### ISSUE TITLE
screenshot should be centered

### ISSUE THREAD

**Issue by philipzae** [solus-project/solus-sc#120]
Would be useful if the screenshot and the screenshot thumbnail list were centered in the window rather than aligned to the left.

![screenshot not centered](https://cloud.githubusercontent.com/assets/4412114/26749659/ab9fa5b8-4820-11e7-8316-c27b499286d7.png)

**Comment by ikeydoherty** [comment]
Fwiw the contents of the entire body need to be centered and allow some stretch room. The layout right now is bork and I'm going off the sidebar idea altogether

**Comment by baimafeima** [comment]
> I'm going off the sidebar idea altogether

What about using the sidebar for the main categories (e.g. Internet, Games, Office, Security, System, etc.)?

**Comment by ikeydoherty** [comment]
Not sure if you read the bit where I said "I'm going off the sidebar idea **altogether**" :P

**Comment by baimafeima** [comment]
> I'm going off the sidebar idea altogether

Yeah I know, just tossing around with ideas. Consider it the last attempt to save the sidebar before it's gone forever. The sidebar in deepin's software center was not bad. Probably better to get rid of it as you say.
